maps/torch/torch.dm

```diff
@@ -174,6 +174,7 @@
 	#include "datums\music\lonely_satelite.dm"
 	#include "datums\music\into_hyperspace.dm"
 	#include "items\hearth_clothing\hats_general.dm"
+	#include "..\..\mods\species\skrell\_skrell.dme"
```

Comment:
```suggestion
```
remove; not ready to include skrell until #20 is merged (i know you included it for testing/CI, though)